src/Token/Token.php

```diff
@@ -41,6 +41,7 @@ final class Token
     public const COMMENT_TAB_TYPE = 'COMMENT_TAB_TYPE';
     public const COMMENT_EOL_TYPE = 'COMMENT_EOL_TYPE';
     public const COMMENT_END_TYPE = 'COMMENT_END_TYPE';
+    public const NAMED_ARGUMENT_OPERATOR_TYPE = 'NAMED_ARGUMENT_OPERATOR_TYPE';
```

Comment:
I'm open to better/shorter name @smnandre

Reply:
NAMED_ARGUMENT_OPERATOR feels not bad (maybe without the "type") ?

Reply:
> (maybe without the "type") ?

So far I suffixed all the tokens by TYPE (like twig did)... So I kinda have to continue

Reply:
Ho sorry :) I'm switching between a few projects i'm losing my mind haha